build.gradle.kts

```diff
@@ -128,6 +128,7 @@ subprojects {
     extra["bytemanVersion"] = "4.0.15"
     extra["commonsBeanVersion"] = "1.9.4"
     extra["commonsVersion"] = "2.6"
+    extra["commonsLang3Version"] = "3.5"
```

Comment:
This is an older version, but see note below as to why this version is used.

Reply:
I see both 3.5 and 3.7 on the test classpaths (compile/runtime)

Reply:
I was stating the version because as a rule I suppose we are not supposed to rely on transitive dependencies in case they change out from under us.

I forgot how to get the exact dependency tree with gradle, but when I stepped into the class with IDEA, it showed the 3.5 jar. However, if we were to state it anyway, why not use the current latest version? It's just a test dependency anyway so it won't affect users.

Reply:
TBH we are probably using lots of transitive without realising! We don't have a similar plugin to what we have in maven to enforce explicitly mentioning it. Honestly, I wouldn't bother adding those lines especially as its just for tests.

Reply:
OK, I will remove the build file changes in that case and just use the classes (already) on the Gradle class path.